sdk/test-utils/recorder-new/README.md

```diff
@@ -12,12 +12,16 @@ Feature work is being tracked at [#15829](https://github.com/Azure/azure-sdk-for
 
 Run this command
 
-> `docker run -v temp-location:/etc/testproxy -p 5001:5001 -p 5000:5000 azsdkengsys.azurecr.io/engsys/testproxy-lin:latest`
+> `docker run -v /workspaces/azure-sdk-for-js/:/etc/testproxy -p 5001:5001 -p 5000:5000 azsdkengsys.azurecr.io/engsys/testproxy-lin:latest`
+
+[Provide the root of the repo for the volume mapping for the accurate location while generating recordings.]
 
 (Eventually, recorder will trigger this for you!)
 
 [Note: Update `temp-location` in the command to your desired location.]
 
+Add `--net=host` to let the Docker container access the localhost on host network.
```

Comment:
```suggestion
Add `--net=host` to disable network sandboxing and let the test proxy container access the host's network (for example, to access `localhost`).
```

This is considered a privileged mode of operation for linux containers, so it should be clear that this disables network namespacing.

Reply:
`--net=host` doesn't work on windows.
Switching to `host.docker.internal` which works on windows and mac(to access host's network) by default and by adding the host in linux with `--add-host host.docker.internal:host-gateway`.

Reply:
My comment is more about making it clear what `--net=host` does from a computer privileges standpoint. The sentence mentioned `--net=host` before, so I just think that if you mention to use it it should also mention that this disables the network namespace entirely and lets the container freely talk to everything on your network.

Reply:
I agree, I did apply your suggestion. :)
But upon more testing, realized that whatever I wrote here didn't work on Windows and had to look for different ways.